include/preprocess/max_inscribed_ellipsoid.hpp

```diff
@@ -140,7 +151,7 @@ std::pair<std::pair<MT, VT>, bool> max_inscribed_ellipsoid(MT A, VT b, VT const&
         YQ.noalias() = Y * Q;
         G = YQ.cwiseProduct(YQ.transpose());
         y2h = 2.0 * yh;
-        YA.noalias() = Y * A;
+        YA = Y * A;
```

Comment:
Since Y is a diagonal matrix, could this operation be further optimized?

Reply:
same as above.